src/bench/crypto_hash.cpp

```diff
@@ -192,10 +192,14 @@ static void SHA512(benchmark::Bench& bench)
 
 static void SipHash_32b(benchmark::Bench& bench)
 {
-    uint256 x;
-    uint64_t k1 = 0;
+    FastRandomContext rng(true);
+    auto k0 = rng.rand64(), k1 = rng.rand64();
+    auto val = rng.rand256();
     bench.run([&] {
-        *((uint64_t*)x.begin()) = SipHashUint256(0, ++k1, x);
```

Comment:
This is a great point. Instead of the type-pun, we'd likely want to make this safer by assigning the result to a `uint64_t` then memcpy it in instead. That likely isn't going to be significantly slower.

Reply:
Do you have any concrete recommendation that I could apply here?